File: FLM_SubconLabelSystem/Library/Library.Database/BLL/CHECK_LOTNO_DUP.vb
Namespace BLL

    Public Class CHECK_LOTNO_DUP

        Public Shared Function check_lotno_dup(ByVal pstr_CompanyCode As String, ByVal pstr_LotNo As String) As String

            Using _dal As New DAL.CHECK_LOTNO_DUP

                Return _dal.check_lotno_dup(pstr_CompanyCode, pstr_LotNo)

            End Using

        End Function

    End Class

End Namespace

File: FLM_SubconLabelSystem/Library/Library.Database/BLL/CHECK_LOTNO_DUP.cs
namespace Library.Database.BLL
{
    public class CHECK_LOTNO_DUP
    {
        public static string check_lotno_dup(string pstr_CompanyCode, string pstr_LotNo)
        {
            using (var _dal = new DAL.CHECK_LOTNO_DUP())
            {
                return _dal.check_lotno_dup(pstr_CompanyCode, pstr_LotNo);
            }
        }
    }
}
